apps/website/src/app/components/icons/TwitterIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export const TwitterIcon = () => {
```

Comment:
Let's update all of the icons to allow props so we can spread on `className` instead of hardcoding them here.

We can do this in a separate PR that touches all icons.

Reply:
Yes, definitely, but I'd do it either when we need to reuse them in slightly other configuration or near the end of things, when functionalities are ready